### PR TITLE
[Fix]Aliados

### PR DESCRIPTION
**Pull request by LindaNieves23**
Fixed

- position purple shape
- title size
- logo borders

Link Trello:
https://trello.com/c/S62UAyno

### CODE REVIEW

**Review by jrmahechad — APPROVED** [review]

File: src/scss/components/_logosSection.scss [hunk 1]
```diff
@@ -17,10 +17,10 @@
     height: 148.31px;
     position: absolute;
     top: 468.64px;
-    left: 349px;
+    right: -85px;
   }
 
-  @include breakpoint(md) {
+  @include breakpoint(lg) {
```
Comment (JFozzie) [inline src/scss/components/_logosSection.scss:23]: There is an issue between 1024px to 1439px,
you can fix it by changing the breakpoint "lg" to "md"

File: src/scss/components/_logosSection.scss [hunk 2]
```diff
@@ -47,7 +47,7 @@
   padding-top: 16rem;
   padding-bottom: 45.8rem;
 
-  @include breakpoint(md) {
+  @include breakpoint(lg) {
```
Comment (JFozzie) [inline src/scss/components/_logosSection.scss:50]: there is an issue between 1024px to 1439px,
you can fix it by changing the breakpoint "lg" to "md"

File: src/scss/components/_logosSection.scss [hunk 3]
```diff
@@ -114,7 +115,7 @@
       border: var(--grey-5) solid 0.1rem;
     }
 
-    @include breakpoint(md) {
+    @include breakpoint(lg) {
```
Comment (JFozzie) [inline src/scss/components/_logosSection.scss:118]: there is an issue between 1024px to 1439px,
you can fix it by changing the breakpoint "lg" to "md"

**Comment by JFozzie** [comment]
The bottom padding in the mobile version of sponsors-section should be 160px

**Review by jsebax — APPROVED** [review]

**Pull request merged**